openapi/openapi_V1.yaml

```diff
@@ -0,0 +1,46 @@
+swagger: '2.0'
+info:
+  title: Microservice JAVA API
+  description: Microservice JAVA API
+  version: '0.0.1'
+host: localhost
+basePath: /api/v1/payment
+schemes:
+  - https
+
+tags:
+  - name: API  Payment
+    description: API  Payment
+
+paths:
+  /check/{noticeNumber}:
+    get:
+      tags:
+        - API  Payment
+      summary: Check by notice number
+      operationId: checkAssistenza
+      parameters:
+        - name: noticeNumber
+          in: path
+          required: true
+          description: payment's notice number
+          type: string      
+      responses:
+        200:
+          description: check if the message has been paid
+          schema:
+            $ref: '#/definitions/ResponseCheck'
+definitions:
+  ResponseCheck:
+    type: object
+    additionalProperties:
```

Comment:
Why we are using additionalProperties instead of definign the message reponse?

Reply:
This bug was fixed in the latest commit